cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -1599,5 +1677,31 @@ void ExportedStubFunctions::AddMappings(Engine* engine) const {
   engine->AddGlobalMappingForFunc("gdv_fn_initcap_utf8",
                                   types->i8_ptr_type() /*return_type*/, args,
                                   reinterpret_cast<void*>(gdv_fn_initcap_utf8));
+
+  // gdv_fn_mask_first_n
+  args = {
```

Comment:
Call this args_to_mask_fns. You dont have to reinitialise this in line 1695

Reply:
Sounds good to me